firmware/quadruna/VC/src/app/app_sbgEllipse.c

```diff
@@ -76,3 +103,46 @@ void app_sbgEllipse_broadcast()
     app_canTx_VC_EulerAnglesPitch_set(euler_pitch);
     app_canTx_VC_EulerAnglesYaw_set(euler_yaw);
 }
+
+static VelocityData app_sbgEllipse_calculateVelocity()
+{
+    // These velocity calculations are not going to be super accurate because it
+    // currently does not compute a proper relative y-axis velocity because no yaw rate 
+
+    float wheelRadius = 0.23f; // 18 inch -> 0.46 meters (diameter) -> 0.23 meters (radius)
+
+    float leftMotorRPM = (float)app_canTx_VC_LeftInverterSpeedCommand_get();
+    float rightMotorRPM = (float)app_canTx_VC_RightInverterSpeedCommand_get();
+
+    float leftWheelVelocity = wheelRadius * (leftMotorRPM  * M_PI_F) / (30 * GEAR_RATIO);
+    float rightWheelVelocity = wheelRadius * (rightMotorRPM * M_PI_F) / (30 * GEAR_RATIO);
```

Comment:
There is a macro for this in app_units or was it utils one of those

Reply:
im not using the macro cuz it doesnt have the accurate radius, if you want i can change the macro or just use the macro with the original value

Reply:
wdym accurate radius? r u talking about the 18in thing? i would change it directly in the macro